util/config/gather-cray-prgenv-arguments

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
```

Comment:
We've started using the `.bash` prefix for scripts that are bash. Will you please do that here?